uc_product/uc_product.module

```diff
@@ -161,6 +161,23 @@ function uc_product_feature_access($node) {
     ((user_access('administer product features') || (user_access('administer own product features') && ($user->uid == $node->uid))));
 }
 
+/**
+ * Implements hook_init().
+ */
+function uc_product_init() {
```

Comment:
See my comment in the main issue. It seems like it would be more efficient to do all this inside `uc_store_image_widget_glightbox()` rather than adding a `hook_init()` implementation just for that.